### PR TITLE
Added the like and dislike buttons

### LINKED ISSUE #198: Enable like and dislike buttons as in chat.susi.ai
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
There are no like and dislike buttons to get response from the user about a particular message.
image

**Expected behavior:**

As thre are like and dislike buttons that record user responses from the user, the same should be implemented in the extension.
image

**Steps to reproduce:**

Open the extension in chrome and type in a message.

**System information:**

```
Your operating system: Windows
```

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #198 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added like and dislike buttons

#### Changes proposed in this pull request:

screen shot 2018-01-17 at 9 18 04 pm

### CODE REVIEW

**Comment by ms10398** [comment]
Travis and codacy checks are failing please correct them

File: src/script.js [hunk 1]
```diff
@@ -150,10 +150,14 @@ function composeReplyTable(response, columns, data) {
 
 function composeSusiMessage(response) {
     var newP = document.createElement("p");
+    var iDiv = document.createElement("div");
```
Comment (gabru-md) [inline src/script.js]: iDiv is created but never used ?

**Comment by stealthanthrax** [comment]
@ms10398 @gabru-md , have a look

**Comment by ms10398** [comment]
Is there any response sent to the server through it?

**Comment by stealthanthrax** [comment]
No,I thought I was not supposed to do that. But I’ll do it now

**Comment by stealthanthrax** [comment]
@ms10398 , have a look now

File: src/style.css [hunk 2]
```diff
@@ -22,7 +22,25 @@ body {
     text-align: center;
     color: white;
 }
-
+#thumb_up{
+        font-size: 15px;
+    }
+    
+    #thumb_up:hover {
```
Comment (gabru-md) [inline src/style.css]: Indentation here please

**Comment by stealthanthrax** [comment]
@ms10398 @gabru-md , have a look now

File: src/style.css [hunk 3]
```diff
@@ -22,7 +22,25 @@ body {
     text-align: center;
     color: white;
 }
-
+#thumb_up{
```
Comment (zamhaq) [inline src/style.css]: inconsistencies in indentation. Please leave a blank after each close `}`

File: src/style.css [hunk 4]
```diff
@@ -22,7 +22,25 @@ body {
...
+    font-size: 15px;
+}
+    
+#thumb_up:hover {
+    color: deepskyblue;
+    cursor: pointer;
+}
+    
+
```
Comment (zamhaq) [inline src/style.css]: only one here

**Comment by stealthanthrax** [comment]
done @zamhaq ,have a look now

**Comment by stealthanthrax** [comment]
@ms10398 @gabru-md @arnav-mandal1234 @zamhaq , have a look

**Comment by gabru-md** [comment]
@ms10398 I strongly oppose this to be merged.
But as it has been merged I would like it to be reverted.
As the PR is wrong.
I would like 2 days for sending a correct PR for the same.
I can explain if you guys want

**Comment by gabru-md** [comment]
@VishalSharma0309 @ms10398 can you guys please give me the reason to approve this PR?

**Comment by gabru-md** [comment]
1. Wrong API is called
2. Wrong data is being send to the server
3. `accessToken` cannot be retrieved through this code. it must be `obj.loggedUser.accessToken`
4. This cannot be implemented without storing the skill associated with the response.
5. Code Redundancy even if I do not consider the above points.

I will work on this as said and send a PR in 2 days.

**Pull request merged**